pkg/reconciler/taskrun/resources/apply.go

```diff
@@ -41,30 +41,37 @@ func ApplyParameters(spec *v1beta1.TaskSpec, tr *v1beta1.TaskRun, defaults ...v1
 	stringReplacements := map[string]string{}
 	arrayReplacements := map[string][]string{}
 
+	patterns := []string{
+		"params.%s",
+		"params[%q]",
+		// FIXME(vdemeester) Remove that with deprecating v1beta1
+		"inputs.params.%s",
```

Comment:
Did I wrote that FIXME ? I think I meant "with deprecating v1alpha1", maybe ? 🤔 👴🏼

Reply:
Yeah, I just moved it 😁

Reply:
yeah.. I saw that 😝 I am confused by my own comment 😹 damn…